kbase-extension/static/kbase/js/widgets/function_output/modeling/kbaseTabTable.js

```diff
@@ -185,23 +182,23 @@ define([
             }
 
             // takes table spec and prepared data, returns datatables settings object
-            this.getTableSettings = function (tab, data) {
+            this.getTableSettings = function (tab) {
                 const tableColumns = getColSettings(tab);
 
                 const settings = {
                     dom: '<"top"lf>rt<"bottom"ip><"clear">',
                     aaData: self.obj[tab.key],
                     aoColumns: tableColumns,
-                    language: { search: '_INPUT_', searchPlaceholder: 'Search ' + tab.name },
+                    language: {search: '_INPUT_', searchPlaceholder: 'Search ' + tab.name},
```

Comment:
looks like this is a DataTables table -- can you update it to use the modern API? `aaData` and `aoColumns` are no longer valid -- I think the modern versions are `data` and `columns`, but you'd have to check.